src/components/tab-bar/examples/tab-bar.tsx

```diff
@@ -60,6 +60,11 @@ export class TabBarExample {
             iconColor: 'var(--lime-deep-red)',
             badge: 4,
         },
+        {
+            id: 8,
+            text: 'Inception',
+            badge: 'Movie',
```

Comment:
I think in a generic component library like Lime Elements, there is no point in limiting the consumers to only send numbers into the tabs. 

That’s rather a design question for the consumer. They need to decide if showing words in the badges is a good idea or not. We standing here shouldn’t care about that. 

But we should perhaps have a "recommendation" note about it. I just don't know currently what to say, to tell users when to use this or not.

So from my point of view, this is PR could be approved already.

Can you just change the word `Movie` to `NEW`. Then the coexistence of those digits and this word would make more sense. It will otherwise be a strange example.

Reply:
⚡